tools/mono-hang-watchdog/mono-hang-watchdog.c

```diff
@@ -0,0 +1,50 @@
+/* Given a external process' id as argument, the program waits for a set timeout then attempts to abort that process */
+/* Used by the Mono runtime's crash reporting. */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <errno.h>
+#include <unistd.h>
+
+#include "config.h"
+
+#ifdef HAVE_SIGNAL_H
+#include <signal.h>
+#endif
+
+#define TIMEOUT 30
```

Comment:
Can we use `TIMEOUT_CRASH_REPORTER_FATAL` for this define?  And can we get it from configure.ac?

Reply:
`TIMEOUT_CRASH_REPORTER_FATAL` had this one use, so it's actually removed with this patch.

I am not sure we use `configure.ac` for any other such overrides of integer values, if I am understanding you.

Reply:
Yea, that's a fair point - maybe this doesn't need to be in configure.

I weakly prefer a longer name for the define - I'm concerned a little bit that it'll be difficult to find the place where this is hardcoded - `TIMEOUT_CRASH_REPORTER_FATAL` is maybe a bit easier to grep than just a generic `TIMEOUT` - not sure.